mvpinternshipapp.client/src/components/product/ProductDetails.jsx

```diff
@@ -0,0 +1,26 @@
+import { Input, Label } from "semantic-ui-react";
+import PropTypes from 'prop-types';
+
+export function ProductDetails({ state, setState }) {
+    return (
+        <div className="detail-grid">
+            <Label as='p'>Name</Label>
+            <Input
+                defaultValue={state.name}
+                onChange={(e) => setState((product) => ({ ...product, name: e.target.value }))}
```

Comment:
These onChange events are fine having this state managements. However, please be aware that if you are going to add some business logic, it will be better if you will create a separate function so that your code will be easier to read and maintain